composer.json

```diff
@@ -13,11 +13,17 @@
 	},
 	"require-dev": {
 		"wp-coding-standards/wpcs": "^1",
-		"phpunit/phpunit": "^6"
+		"phpunit/phpunit": "^6",
+		"yoast/phpunit-polyfills": "^1.0"
```

Comment:
If you are adding the PHPUnit Polyfills, it's better to remove the `phpunit/phpunit` requirement in favour of letting the Polyfills manage the supported PHPUnit versions.

(aside from that, this doesn't look like change which belongs in this PR ?)

Reply:
Good catch. Thanks.